crest/Assets/Crest/Crest/Scripts/LodData/RegisterClipSurfaceInput.cs

```diff
@@ -179,7 +214,11 @@ public override void Draw(LodDataMgr lodData, CommandBuffer buf, float weight, i
             buf.SetGlobalFloat(LodDataMgr.sp_LD_SliceIndex, lodIdx);
             buf.SetGlobalVector(sp_DisplacementAtInputPosition, Vector3.zero);
 
-            if (_mode == Mode.Primitive)
+            if (_inputMode == InputMode.Painted)
+            {
+                buf.DrawProcedural(Matrix4x4.identity, _paintInputMaterial, 0, MeshTopology.Triangles, 3);
```

Comment:
Loading the main scene this is throwing exception because the material doesn't exist.

Reply:
fixed, thanks